balancer-js/src/modules/pools/pool-types/concerns/weighted/liquidity.concern.ts

```diff
@@ -16,7 +16,14 @@ export class WeightedPoolLiquidity implements LiquidityConcern {
         continue;
       }
 
-      const price = parseFixed(token.price.usd.toString(), SCALING_FACTOR);
+      let price: BigNumber;
+      if (parseFloat(token.price.usd) < 1) {
```

Comment:
This could be a problem for other pool types too right? Could it be pulled out into a separate function so it can be re-used. Also could we add a test to ensure it fixes the issue and calculates liquidity correctly for that pool.